prowler/providers/aws/services/eks/eks_cluster_uses_a_supported_version/eks_cluster_uses_a_supported_version.py

```diff
@@ -0,0 +1,28 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.eks.eks_client import eks_client
+
+
+class eks_cluster_uses_a_supported_version(Check):
+    def execute(self) -> Check_Report_AWS:
+        findings = []
+
+        for cluster in eks_client.clusters:
+            report = Check_Report_AWS(self.metadata())
+            report.status = "PASS"
+            report.region = cluster.region
+            report.resource_id = cluster.name
+            report.resource_arn = cluster.arn
+            report.resource_tags = cluster.tags
+            report.status_extended = f"EKS cluster {cluster.name} is using version {cluster.version} that is supported by AWS."
+
+            eks_supported_versions = eks_client.audit_config.get(
```

Comment:
I would prefer having a variable like in Security Hub like `oldestVersionSupported `

Reply:
Sure what does @puchy22 think about this? Because originally I had it has oldest version 1.28, and then in the check, I just looked for if the users current version was above or below 1.28.

Reply:
Ok sounds good, so just have a single value in the config file, and then in the check, just see if the users version of Kubernetes is higher to lower than that version?